src/client/components/meal/MealsPage.js

```diff
@@ -0,0 +1,69 @@
+import React, { useEffect, useState } from "react";
+import "./meal.css";
+
+import AddMeal from "./AddMeal";
+import MealLink from "./MealLink";
+
+import Header from "../menu/Header";
+import Modal from "../modal/Modal";
+
+function MealsPage() {
+  const [meals, setMeals] = useState([]);
+  const [permited, setPermited] = useState({});
+
+  const [isOpen, setIsOpen] = useState(false);
+  useEffect(() => {
+    getMeals();
+  }, []);
+  const getMeals = async () => {
+    try {
+      const response = await fetch("/api/meals");
+      const results = await response.json();
+
+      setMeals(results);
+    } catch (err) {
+      throw new Error("fetching meals went wrong");
```

Comment:
I'd expect the page to not crash because of that, let's show an error state to the user instead.